templates/components/user/info_card.html.twig

```diff
@@ -59,10 +59,18 @@
                   <a href="mailto:{{ user['email'] }}">{{ user['email'] }}</a>
                </div>
             {% endif %}
-            {% if user['phone']|length > 0 %}
+            {% if not user['phone'] is empty or not user['phone2'] is empty %}
                <div>
                   <i class="fas fa-fw fa-phone"></i>
-                  <a href="tel:{{ user['phone'] }}">{{ user['phone'] }}</a>
+                  {% if not user['phone'] is empty %}
+                     <a href="tel:{{ user['phone'] }}">{{ user['phone'] }}</a>
+                  {% endif %}
+                  {% if not user['phone'] is empty and not user['phone2'] is empty %}
+                     <span> - </span>
+                  {% endif %}
+                  {% if not user['phone2'] is empty %}
+                     <a href="tel:{{ user['phone2'] }}">{{ user['phone2'] }}</a>
+                  {% endif %}
```

Comment:
I think it wwould be better to have the phone icon for both phone numbers.
```
               <div>
                  {% if not user['phone'] is empty %}
                     <i class="fas fa-fw fa-phone"></i>
                     <a href="tel:{{ user['phone'] }}">{{ user['phone'] }}</a>
                  {% endif %}
                  {% if not user['phone'] is empty and not user['phone2'] is empty %}
                     <span> - </span>
                  {% endif %}
                  {% if not user['phone2'] is empty %}
                     <i class="fas fa-fw fa-phone"></i>
                     <a href="tel:{{ user['phone2'] }}">{{ user['phone2'] }}</a>
                  {% endif %}
               </div>
```

Reply:
Done:   

![image](https://user-images.githubusercontent.com/42734840/190131457-bf444a39-e379-4e45-b47a-ea5a61c8af5c.png)